src/app/shared/pipes/unit-filter.pipe.ts

```diff
@@ -65,6 +67,22 @@ export class UnitFilterPipe implements PipeTransform {
           break;
       }
     }
+    if (arg.abilityTargetType) {
+      switch (arg.abilityTargetType) {
+        case 'crew':
+          filtered = filtered.filter(u => this.targetsCrew(u.lvl5Ability));
+          break;
+        case 'type':
+          filtered = filtered.filter(u => this.targetsTypes(u.lvl5Ability, u.type));
```

Comment:
I think there are some abilities which target multiple types. Particularly, on the buff side

Reply:
that wouldn't be a problem but it got me wondering about abilities which buff other types/classes besides the unit own type/classes. i think it also wouldn't work for dual unit as it was. now i double checked and all cases seem to be working fine. thanks